src/components/settings/ReportsTab.vue

```diff
@@ -66,6 +118,22 @@ export default {
     handleConfig() {
       this.$emit("submit-config", this.config);
     },
+
+    async openImage() {
+      if (this.$isElectron) {
+        const { status, message, path } =
+          await this.$electronService.openImage();
+
+        if (status === STATUSES.ERROR) {
+          this.$root.$emit("set-snackbar", message);
+        } else {
+          this.config.logoPath = path;
+          this.$emit("submit-config", this.config);
+        }
+      } else {
+        // todo Add web version handler
+      }
```

Comment:
@dockabernathy01 - If we just use a standard [`input`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file) for this, then we can get the file path directly from the that element _and_ it will work for both web and electron.  Then we can remove all of the `$electronService` specific code.

Reply:
I'm sorry. it will work on web version too.
I couldn't remove that.
It works.

Reply:
I would like this done with a standard HTML file input.

Reply:
Currently, I used electron dialog.
You mean, you want me to use v-file input?

Reply:
Like this.
const {  dialog,  BrowserWindow } = require("electron");
dialog.showOpenDialog()

Reply:
Yes, please use a `v-file-input` in the view above and have the image card src display the value of that input.

Reply:
Ok, I will change like that.
But I think, it doesn't look good in ui side.

Reply:
And in addition, why don't you use Slack or MS Teams for discussing about the work?(Of course, we will do contract on upwork.)
I think it's more flexible to discuss about issues.